packages/kbn-test/src/functional_tests/lib/log.js

```diff
@@ -0,0 +1,4 @@
+import { createToolingLog } from '@kbn/dev-utils';
+
+export const log = createToolingLog('debug');
```

Comment:
IMO, the CLI functions should create a `log` instance that is passed down from there to any function that needs it. Then we can get rid of this module scope cache and allow [configuring the log level](https://github.com/elastic/kibana/blob/master/packages/kbn-dev-utils/src/tooling_log/log_levels.js#L3-L9)